_build/templates/default/package.json

```diff
@@ -1,25 +1,27 @@
 {
   "name": "revolution-theme-default",
   "title": "MODX Revolution",
-  "version": "2.3.0",
+  "version": "2.7.0",
   "repository": "https://github.com/modxcms/revolution",
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-autoprefixer": "^3.0.3",
-    "grunt-bower-task": "^0.4.0",
-    "grunt-contrib-clean": "^0.5.0",
-    "grunt-contrib-copy": "^0.6.0",
-    "grunt-contrib-concat": "^0.5.1",
-    "grunt-contrib-csslint": "^0.4.0",
-    "grunt-contrib-cssmin": "^0.13.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-contrib-uglify": "^0.9.2",
+    "bourbon": "^4.3.4",
+    "font-awesome": "^4.7.0",
+    "grunt": "^1.0.3",
+    "grunt-autoprefixer": "^3.0.4",
+    "grunt-bower-task": "^0.5.0",
```

Comment:
I guess bower task also should be removed.

Reply:
Yep.